Indicators/readme.md

```diff
@@ -0,0 +1,290 @@
+<h2>Guide to Developing Indicators in Lean</h2>
```

Comment:
Please remove the doc from this PR and we can draft offline and submit to the wiki or the tutorials repo.

Reply:
Removed!